esmvaltool/utils/recipe_test_workflow/recipe_test_workflow/flow.cylc

```diff
@@ -31,7 +31,7 @@
         env-script = "eval $(rose task-env)"
         [[[environment]]]
             ENV_NAME = {{ ENV_NAME }}
-            USER_CONFIG_DIR = ${CYLC_WORKFLOW_RUN_DIR}/etc
+            USER_CONFIG_DIR = ${CYLC_WORKFLOW_SHARE_DIR}/cycle/${CYLC_TASK_CYCLE_POINT}
```

Comment:
`${ROSE_DATAC}` can be used here instead of `${CYLC_WORKFLOW_SHARE_DIR}/cycle/${CYLC_TASK_CYCLE_POINT}` 👍

Reply:
Fixed in commit [8d01587](https://github.com/ESMValGroup/ESMValTool/pull/3600/commits/8d01587e4901648286618161b6eafbb1d92a6fe4)